From 2554fc24297f55ba95039e09cd9d074f82418298 Mon Sep 17 00:00:00 2001
From: Varadarajan Narayanan <quic_varada@quicinc.com>
Date: Tue, 7 Jan 2025 21:44:02 +0530
Subject: [PATCH 15/20] PENDING: arm64: dts: qcom: qcs9100: Add missed DDR
 memory region

Added the DDR memory region that was missed earlier.

Signed-off-by: Varadarajan Narayanan <quic_varada@quicinc.com>
Upstream-Status: Pending
---
 dts/upstream/src/arm64/qcom/qcs9100-ride-r3.dts | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/dts/upstream/src/arm64/qcom/qcs9100-ride-r3.dts b/dts/upstream/src/arm64/qcom/qcs9100-ride-r3.dts
index f0453315a0..ce92d4a241 100644
--- a/dts/upstream/src/arm64/qcom/qcs9100-ride-r3.dts
+++ b/dts/upstream/src/arm64/qcom/qcs9100-ride-r3.dts
@@ -19,6 +19,7 @@
                         <0x9 0x00000000 0x1 0x80000000>,
                         <0x1 0x00000000 0x2 0xf7500000>,
                         <0x0 0xd0000000 0x0 0x00100000>,
-                        <0x0 0xd3500000 0x0 0x07c00000>;
+                        <0x0 0xd3500000 0x0 0x07c00000>,
+                        <0x0 0xdb300000 0x0 0x24d00000>;
         };
 };
-- 
2.34.1

